src/app/datagrid/selection/selection.html

```diff
@@ -112,7 +120,11 @@ <h3>Client side, trackBy item</h3>
     <ng-container *clrDgHideableColumn> Favorite color </ng-container>
   </clr-dg-column>
 
-  <clr-dg-row *clrDgItems="let user of clientTrackByIdUsers; trackBy: trackById" [clrDgItem]="user">
+  <clr-dg-row
+    *clrDgItems="let user of clientTrackByIdUsers; trackBy: trackById"
+    [clrDgItem]="user"
+    [clrDgRowAriaLabel]="'Select row for ' + user.name"
```

Comment:
This updates the demos on the angular docs application to create unique user names.